Sources/Mantis/CropView/CropView.swift

```diff
@@ -956,7 +970,27 @@ extension CropView: CropViewProtocol {
         viewModel.fixedImageRatio = -1
     }
     
+    public func applyCropState(with cropState: CropState) {
+        viewModel.rotationType = .none
+        viewModel.horizontallyFlip = cropState.transformation.horizontallyFlipped
+        viewModel.verticallyFlip = cropState.transformation.verticallyFlipped
+        viewModel.fixedImageRatio = cropState.aspectRato
+        flipOddTimes = cropState.flipOddTimes
+        
+        let newTransform = getTransformInfo(byTransformInfo: cropState.transformation)
+        
+        // The first transform is just for retrieving the final cropBoxFrame
+        transform(byTransformInfo: newTransform, isUpdateRotationControlView: true)
```

Comment:
I checked restore transform logic again. Looks like we can just remove this first `transform(byTransformInfo:`. The reason I used two `transform(byTransformInfo:` functions is because I need to retrieve the final cropBoxFrame in case of applying to different devices.
I did some simple tests locally after deleting this line.

Can you also help to do a test too to see if it works after removing this line? Thank you!

Reply:
I will test it this evening. Earlier, when I tried to remove it some transforms failed. Perhaps I tested it before adding more state to CropState struct.

Reply:
Removing first call fails if you:

1. Rotate Left (90 CCW)
2. Undo

Reply:
Perhaps refactor logic needed into separate function and call that before second transform and inside transform.

Reply:
Yes now I see it fails in Catalyst. OK, then just keep the current logic for now.
I may refactor the logic in the future.

Reply:
@rickshane 
Can you replace the first `transform(byTransformInfo:` with the code below
I did a little tests which works fine.

```Swift
if newTransform.maskFrame != .zero 
     viewModel.cropBoxFrame = newTransform.maskFrame
}
```

Reply:
That seems to work.

Reply:
change made